website/docs/addon_maya_ornatrix_artist.md

```diff
@@ -98,3 +68,18 @@ The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix dat
 ### Loading
 
 The **Ornatrix Cache** supports to load Ornatrix alembic (.abc), Vray Proxy (.abc) and Arnold Scene Source (.abc).
+
+## FAQ
+### How to connect an Ornatrix rig with a pointcache or animation?
+After loading the Ornatrix Rig and a pointcache to connect it to, you can use the AYON tools to connect it for you:
```

Comment:
```suggestion
Once you’ve loaded the Ornatrix Rig and its corresponding pointcache, you can connect them using the AYON tools: 
```

Reply:
what about 
```suggestion
Once you’ve loaded the Ornatrix Rig and its corresponding pointcache, you can connect them using AYON Inventory Actions: 
```

Reply:
We should use you have instead of you've.

Reply:
what about: 
```suggestion
Once you load the Ornatrix Rig along with its corresponding pointcache, you can link them using AYON Inventory Actions:
```

Reply:
I've applied the last suggestion. feel free to unresolve.
https://github.com/ynput/ayon-documentation/pull/237/commits/5df28b622d2dbf737f57502e6b492bda2b003790